nil/services/rpc/jsonrpc/send_transaction.go

```diff
@@ -19,12 +22,33 @@ func (api *APIImpl) SendRawTransaction(ctx context.Context, encoded hexutil.Byte
 
 	shardId := extTxn.To.ShardId()
 	reason, err := api.rawapi.SendTransaction(ctx, shardId, encoded)
+
+	headers, ok := ctx.Value(transport.HeadersContextKey).(http.Header)
+	if !ok {
+		headers = http.Header{}
+		headers.Add("Client-Type", "failed to extract headers from context")
+	}
+
+	// even though warn here looks weird these logs shouldn't be
+	// occasionally gone when global log level is increased
+	log := api.clientEventsLog.Warn().
+		Stringer(logging.FieldShardId, shardId).
+		Str(logging.FieldRpcMethod, "eth_sendRawTransaction").
+		Str(logging.FieldClientType, headers.Get("Client-Type")).
+		Str(logging.FieldClientVersion, headers.Get("Client-Version")).
+		Str(logging.FieldUid, headers.Get("X-UID")) //nolint:canonicalheader
+
 	if err != nil {
+		log.Msg("finished with err")
```

Comment:
Usually we add error itself
```suggestion
		log.Err(err).Msg("finished with err")
```